Sources/ConsoleKit/Command/Completion.swift

```diff
@@ -297,7 +303,7 @@ extension CompletionAction {
     /// Creates a `CompletionAction` that provides a predefined list of possible values.
     public static func values(_ values: [String]) -> CompletionAction {
         return [
-            .bash: "COMPREPLY=( $(compgen -W \"\(values.joined(separator: " "))\" -- $cur) )",
+            .bash: "COMPREPLY+=( $(compgen -W \"\(values.joined(separator: " "))\" -- $cur) )",
```

Comment:
The completion values for label-less `Argument`s are _appended_ to `COMPREPLY`, which will already contain any `Option` or `Flag` labels.